modules/component/src/push/push.pipe.ts

```diff
@@ -67,63 +54,46 @@ import {
  */
 @Pipe({ name: 'ngrxPush', pure: false })
 export class PushPipe<S> implements PipeTransform, OnDestroy {
-  private renderedValue: S | null | undefined;
-
-  private readonly configSubject = new Subject<PushPipeConfig>();
-  private readonly config$ = this.configSubject
-    .asObservable()
-    .pipe(distinctUntilChanged());
+  private renderedValue: any | null | undefined;
 
   private readonly subscription: Unsubscribable;
   private readonly cdAware: CdAware<S | null | undefined>;
-  private readonly updateViewContextObserver: PartialObserver<
+  private readonly resetContextObserver: NextObserver<unknown> = {
```

Comment:
Honestly, I wouldn't use RxJS's types here, just let TS figure it out. We may remove `NextObserver` at some point

Reply:
I remember. There is the potential deprecation of callbacks.
I will definitely do it.

Reply:
Ok, @benlesh I tried and I realized I don't know how... I can't not type it (implicit any).

I can just assign the function in the pipe without typing:
Before:
```typescript
 private readonly resetContextObserver: NextObserver<unknown> = {
    next: (value: unknown) => (this.renderedValue = undefined),
 };
```
After:
```typescript
 private readonly resetContextObserver = (value: unknown) => (this.renderedValue = undefined);
```

But in the CdAware creating function I have to type it, right??
```typescript
export function createCdAware<U>(cfg: {
  strategies: StrategySelection<U>;
  resetContextObserver: NextObserver<any>;
  updateViewContextObserver: PartialObserver<U> & NextObserver<U>;
})
```

Reply:
I was not able to have it working other than introducing my own typing for the same thing.

Keep it for now.